### DISCUSSION TITLE
Unable to Synchorize iModel

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#4767] [General]
I am unable to start synchronization of my model in iTwin Synchronizer.  When I start synchronization process, I get the following message (this is new - I didn't get this before)

![alert](https://user-images.githubusercontent.com/83007711/205964848-361c0310-892b-43f5-a452-1cc54ad3ea35.JPG)

When I press Confirm, it takes me to a browser window. 
 
![image](https://user-images.githubusercontent.com/83007711/205965109-02dcde3d-d64a-46f0-a85f-fd0042ae0371.png)

I close the browser as it says in the image, the control goes back to iTwin Synchronizer dialog and nothing happens.  Pressing Synchronize again, the alert comes up again and same repeats.

It has been couple of weeks since last time I used iTwin Synchronizer, this alert and confirmation request did not come up before.  Though when I started synchronization process today, it first updated 5-6 connectors before I could even press Synchronization button.

Can you please advise what I need to do to get synchronization working again

**Comment by jffmarker** [comment]
CC: @Sneha26115 from the iTwin Synchronizer team.

Unless Sneha has any thoughts off-hand, I think we'd need to collect the logs to better diagnose. When you start the iTwin Synchronizer client, click the gear icon in the upper-right and select Save Logs. You may wish to open a [service ticket](https://apps.bentley.com/srmanager/) to provide the logs to keep them more confidential.

**Reply by ovaisa** [reply]
Seems to work today - this alert didn't come up and was able to synchronize model successfully.  Not sure what was different as everything is same as yesterday...

While I am on this subject of iTwin Synchronizer, I would like to request Bentley to consider not automatically updating connectors. It always start when one can least afford.  Yesterday I had to download 3GB worth of connectors and it took a better half of my day to get it working again - while client was waiting for an update...  

I am afraid this update of connectors is never a smooth process (at least for me). Got network error and it was unrecoverable.  Had to use Task Manager to kill the app several time - but it seems the download process continues and restarting app doesn't seem to stop this download.  Had to reboot several times to get this to work.  And I am afraid what happened yesterday is more of a norm then exception.

I realize keeping connectors up to date is important, ideally user should be given option to update them at a time that works better and doesn't get in way of completing work.

Thank you,

Ovais

**Reply by SneIdr19** [reply]
The consent alert was introduced in the latest release and we have some documentation around it in FAQ. I am pasting links here for your reference.
https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/47622/frequently-asked-questions-its 
https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/62729/itwin-synchronizer---latest-version-and-notes-listing

Sorry to hear about trouble with connector updates. If you could file an SR highlighting the problems with update and attach app logs we can investigate further. As for connector auto-updates, they are mandated by design in order to keep the local copies of the connector up to date. We do allow a few days of grace period when these connectors don't force an update but it is a short duration of few days. The concern is if we allow an option to postpone an update, the user may use that choice longer than is advisable leading to other side effects.. You may however make a feature request for this change to be considered.

**Reply by ovaisa** [reply]
I believe some giving option to update within a grace period would work well.  If someone ignores the warning, then I believe forcing the update is fine - this is what Apple does with OS upgrades.  Right now user doesn't have any say and more often that not, it gets in the way of work.  Please consider this as a feature request.

Thank you
